cypress/features/regression/test/actionPopover.feature

```diff
@@ -42,15 +42,15 @@ Feature: Action Popover component in noIFrame
 
   @positive
   Scenario: verify that the first element Business is focused using Home key
-    Given I click the menu button element in noiFrame
+    Given I click the menu button element
       And I press keyboard "downarrow" key times 2
     When I press "Home" onto focused element
     Then focused element inner content is set to "Business"
       And Action Popover element has golden border on focus
 
   @positive
   Scenario: verify that the first sub menu 1 element is focused using Home key for submenu
```

Comment:
Could this scenario and others in the submenu level be put into an Examples table?

Reply:
nope, because those TCs have different steps to proceed